### PR TITLE
Migrate goonchat pings to a subsystem

### PR DESCRIPTION
**Pull request by SpaceManiac**
Requested by @MrStonedOne.

### CODE REVIEW

File: code/controllers/subsystem/ping.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
+SUBSYSTEM_DEF(ping)
+	name = "Ping"
+	priority = FIRE_PRIORITY_PING
+	wait = 30  // one and a half seconds
```
Comment (MrStonedOne) [inline code/controllers/subsystem/ping.dm]: Wait is in deciseconds, not ticks, unless it has the ticker subsystem flag (this does not).

This is 3 seconds of wait.

**Review by MrsTonedOne — APPROVED** [review]

**Comment by optimumtact** [comment]
please replace the stat ping counter with this same subsystem count

**Comment by SpaceManiac** [comment]
The two systems work completely differently, the server doesn't even know the value that's shown in goonchat, though if you wanted I could move the update of that stat from SSserver_maint to here.

**Comment by optimumtact** [comment]
hmm it just bothers me that we have two, do we really need two?

**Comment by Cyberboss** [comment]
As long as we support old chat I assume so

**Pull request merged**